build.gradle

```diff
@@ -39,15 +39,22 @@ group = 'org.python'
 // ---------------- Miscellaneous configuration --------------------------------
 
 allprojects {
+
     apply plugin: 'java'
     // We support Java 11 onwards officially.
     sourceCompatibility = '11'
+
     // Same configuiration all sub-projects
     tasks.withType(JavaCompile) {
         options.encoding = 'UTF-8'
         options.debug = true
```

Comment:
Perhaps you want to set `--release 11` too?

> --release release
> 
> Compiles against the public, supported and documented API for a specific VM version. Supported release targets are 6, 7, 8, 9, 10, and 11.
> 
> Note:
> 
> When using --release for a version of the Java Platform that supports modules, you can’t use --add-modules to access internal JDK modules, nor can you use --add-exports to access internal JDK APIs in the modules.
>
> https://docs.oracle.com/en/java/javase/11/tools/javac.html#GUID-AEEC9F07-CB49-4E96-8BC7-BCC2C7F725C9

Reply:
That seems a useful discipline to accept. The magic is:
```
    tasks.withType(JavaCompile) {
        // Use only public API
        options.compilerArgs.addAll(['--release', '11'])
        ...
    }
```
I'll slip it into `main` later.